.github/ISSUE_TEMPLATE/2_feature_request.yml

```
@@ -0,0 +1,25 @@
+name: 🆕 Suggest an improvement or new feature
+description: ___
+labels: enhancement, under discussion
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Please ensure your requested feature makes sense for Refined GitHub:
+        https://github.com/sindresorhus/refined-github/issues/2960
+  - type: textarea
+    attributes:
+      label: Describe your improvement or feature request.
```

Let's not end field labels with a period. It should probably be a colon but since they're headline-sized they work without anything

```suggestion
      label: Describe your improvement or feature request
```

---

```suggestion
      label: Description
```